### PR TITLE
during bundle generate, ask the user where they want to output the bu…

### PR DESCRIPTION
**Pull request by chrisghill**
…ndle

### CODE REVIEW

File: cmd/application.go [hunk 1]
```diff
@@ -17,15 +17,17 @@ import (
 
 // applicationCmd represents the application command
 var applicationCmd = &cobra.Command{
-	Use:   "application",
-	Short: "Application development tools",
-	Long:  ``,
+	Use:     "application",
+	Aliases: []string{"app"},
```
Comment (chrisghill) [inline cmd/application.go:21]: just adding an alias here, so you can do `mass app gen`

File: pkg/bundle/prompt.go [hunk 2]
```diff
@@ -86,3 +87,19 @@ func getDescription(t *TemplateData) error {
 	t.Description = result
 	return nil
 }
+
+func getOutputDir(t *TemplateData) error {
```
Comment (chrisghill) [inline pkg/bundle/prompt.go:91]: This is the only real change associated with the prompt

File: pkg/bundle/generate.go [hunk 3]
```diff
@@ -1,127 +1,50 @@
 package bundle
```
Comment (chrisghill) [inline pkg/bundle/generate.go:1]: These changes are a red herring.  I got rid of the `generator` package and moved the files into the `bundle` package.  However, both packages had a `generate.go` file. So I moved the `bundle/generate.go` to `bundle/schema.go`, and the `generator/generate.go` became `bundle.generate.go`.

So basically GitHub is throwing a ton of diffs when its a completely different file.

File: pkg/bundle/generate_test.go [hunk 4]
```diff
@@ -1,49 +1,56 @@
 package bundle_test
```
Comment (chrisghill) [inline pkg/bundle/generate_test.go:1]: Same as above. This file is now `bundles/schema_test.go` and `generator/generate_test.go` is this file. The diff here is irrelevant.

File: pkg/bundle/schema.go [hunk 5]
```diff
@@ -0,0 +1,127 @@
+package bundle
```
Comment (chrisghill) [inline pkg/bundle/schema.go:1]: This was `bundle/generator.go`

File: pkg/bundle/schema_test.go [hunk 6]
```diff
@@ -0,0 +1,49 @@
+package bundle_test
```
Comment (chrisghill) [inline pkg/bundle/schema_test.go:1]: This was `generator_test.go`. No changes were made.

File: pkg/bundle/prompt.go [hunk 7]
```diff
@@ -86,3 +87,19 @@ func getDescription(t *TemplateData) error {
...
+	prompt := promptui.Prompt{
+		Label:   `Output directory`,
+		Default: t.Name,
```
Comment (WillBeebe) [inline pkg/bundle/prompt.go:94]: perfect default 👌

**Review by WillBeebe — APPROVED** [review]
LGTM thanks Chris!

**Pull request merged**